ci-operator/step-registry/openshift-power-monitoring/tests/kepler/openshift-power-monitoring-tests-kepler-commands.sh

```diff
@@ -5,6 +5,7 @@ set -eu -o pipefail
 declare -r LOGS_DIR="/$ARTIFACT_DIR/test-run-logs"
 declare -r OPERATOR_DEPLOY_NAME="kepler-operator-controller"
 declare -r OPERATORS_NS="openshift-operators"
+declare -r KEPLER_NS="kepler-operator"
```

Comment:
This should be configurable (may be in another PR). for now, could you please use `${KEPLER_NS}` and use 
`KEPLER_DEPLOYMENT_NS` ?